README.md

```diff
@@ -63,6 +64,8 @@ For example, the `cordova-plugin-device` plugin has this nested [`plugin.xml`](h
 
 The `cordova-plugin-test-framework` plugin will automatically find all `tests` modules across all plugins for which the nested tests plugin is installed.
 
+> **Important:** Inside your project's `tests/` folder you also have to create a `package.json`. See the [`package.json` of the `cordova-plugin-device`](https://github.com/apache/cordova-plugin-device/blob/736c7b9dfdfa25a924ffc3d1e409450633a8c00f/tests/package.json) for an example.
```

Comment:
Ah yes, good point. Could I suggest, however, that we reword this a bit and move the paragraph up above the previous "...will automatically find all `tests` modules" paragraph?

In particular, I would structure this paragraph just like the first paragraph in the section, in a similar, instruction-providing way. Explaining _why_ this needs to be done would be a nice touch, too - it would help users understand the intentions behind each requirement. In particular, the latest version of the tools ensure to run `npm install` on any plugin added to a project, to ensure to pull in any dependencies. Therefore, plugin authors can now put npm dependencies around their tests into the `package.json` file.